lib/core/src/lib/comments/comments.component.ts

```diff
@@ -57,6 +57,10 @@ export class CommentsComponent implements OnChanges {
     @Output()
     error = new EventEmitter<any>();
 
+    /** Emits when a new comment is added */
+    @Output()
```

Comment:
Please add unit tests covering the changes and also update the documentation for the component